README.md

```diff
@@ -1,3 +1,13 @@
+This repo is home to the Uniswap Widgets package and the web app interface [app.uniswap.org](https://app.uniswap.org).
+
+# Uniswap Widgets
+
+The @uniswap/widgets package is an npm package of React components used to provide subsets of the Uniswap Protocol functionality in a small and configurable user interface element.
```

Comment:
The only thing actually exposed for usage right now is the SwapWidget, so I don't think you should promise anything further until it's available.